eng/common/native/CommonLibrary.psm1

```diff
@@ -0,0 +1,389 @@
+<#

```

Comment:
I don't think we need files like `eng/common/native`, `eng/common/internal`, `eng/common/performance`... should we add them to the .gitignore and remove them?